seqr/views/utils/test_utils.py

```diff
@@ -668,6 +668,26 @@ def _get_expected_dag_variables(self, omit_project=None, **kwargs):
         }
 
 
+class AirtableTest(object):
+
+    def assert_expected_airtable_call(self, call_index, filter_formula, fields, additional_params=None):
+        expected_params = {
+            'fields[]': mock.ANY,
+            'pageSize': '100',
+            'filterByFormula': filter_formula,
+        }
+        if additional_params:
+            expected_params.update(additional_params)
+        self.assertDictEqual(responses.calls[call_index].request.params, expected_params)
+        self.assertListEqual(self._get_list_param(responses.calls[call_index].request, 'fields%5B%5D'), fields)
+
+    @staticmethod
+    def _get_list_param(call, param):
```

Comment:
I know this is copy-paste and a test helper so no changes requested here, but we're using a real url parser elsewhere right?

Reply:
yeah this is just for tests